lib/build_targets/application.dart

```diff
@@ -79,7 +77,7 @@ abstract class TizenAssetBundle extends Target {
     final Depfile assetDepfile = await copyAssets(
       environment,
       outputDirectory,
-      targetPlatform: null, // corresponds to flutter-tester
+      targetPlatform: TargetPlatform.android,
```

Comment:
This only affects [SkSL caching](https://github.com/flutter-tizen/engine/issues/99) and will be reworked when needed.